src/core_atmosphere/physics/physics_wrf/module_sf_sfclay.F

```diff
@@ -388,22 +393,32 @@ SUBROUTINE SFCLAY1D(J,UX,VX,T1D,QV1D,P1D,dz8w1d,                &
       REAL    ::  FLUXC,VSGD,Z0Q,VISC,RESTAR,CZIL,GZ0OZQ,GZ0OZT
       REAL    ::  ZW, ZN1, ZN2
       REAL    ::  Z0T, CZC
+!$acc data create(PSFC,TGDSA,THGB,SCR3,THX,SCR4,THVX,QX,ZQKLP1, &
+!$acc ZQKL,ZA,GOVRTH,RHOX,GZ2OZ0,GZ10OZ0,PSIM10,PSIH10,PSIM2, &
+!$acc PSIH2,WSPDI,DENOMQ,DENOMQ2,DENOMT2)
 !-------------------------------------------------------------------
       KL=kte
 
+!$acc parallel vector_length(32)
+!$acc loop gang vector
       DO i=its,ite
 ! PSFC cb
          PSFC(I)=PSFCPA(I)/1000.
       ENDDO
 !                                                      
 !----CONVERT GROUND TEMPERATURE TO POTENTIAL TEMPERATURE:  
 !                                                            
-      DO 5 I=its,ite                                   
+!!!   DO 5 I=its,ite                                   
```

Comment:
I think it would help maintain cleaner code to just delete the old-style `DO` loops rather than keeping them as comments.